### PR TITLE
Feat trans

### PR DESCRIPTION
**Pull request by prtkjakhar**

### CODE REVIEW

File: apps/amakrushi/lang/en.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "label.new_chat":"New chat",
     "label.confirm_delete":"Are you sure you want to delete this conversation?",
     "label.no_history":"No Chats",
+    "label.translate":"Translate",
```
Comment (ChakshuGautam) [inline apps/amakrushi/lang/en.json:25]: Should we rename this to transliterate? It has to be `transliterate` everywhere.

**Review by ChakshuGautam — APPROVED** [review]

File: apps/amakrushi/lang/en.json (hunk 1 above)
Comment (prtkjakhar) [reply apps/amakrushi/lang/en.json]: done

File: apps/amakrushi/lang/en.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "label.new_chat":"New chat",
     "label.confirm_delete":"Are you sure you want to delete this conversation?",
     "label.no_history":"No Chats",
+    "label.translate":"Transliterate",
```
Comment (ChakshuGautam) [inline apps/amakrushi/lang/en.json:25]: label should also change no?

File: apps/amakrushi/src/components/HomePage/index.tsx [hunk 3]
```diff
@@ -67,6 +69,41 @@ const HomePage: NextPage = () => {
...
+            input: input
+          },
+          {
+            headers: {
+              'Content-Type': 'application/json'
+            },
+          }
+        );
+        console.log(response.data.output);
+        const translatedArray = [];
```
Comment (ChakshuGautam) [inline apps/amakrushi/src/components/HomePage/index.tsx:97]: This as well.

Comment (prtkjakhar) [reply apps/amakrushi/src/components/HomePage/index.tsx]: done

File: apps/amakrushi/lang/en.json (hunk 2 above)
Comment (prtkjakhar) [reply apps/amakrushi/lang/en.json]: done

**Pull request merged**